### PR TITLE
Us/cart

### PR DESCRIPTION
**Pull request by markus0059**

### CODE REVIEW

File: src/Controller/CartController.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+namespace App\Controller;
+
+use App\Model\CartManager;
+use App\Model\ProductManager;
+
+class CartController extends AbstractController
+{
+    public function index()
+    {
+        if ($_SESSION["user_id"] === $this->user['id']) {
```
Comment (n3en) [inline src/Controller/CartController.php]:
```suggestion
        if (!is_null($this->user)) {
```

**Review by n3en — CHANGES_REQUESTED** [review]

File: src/View/Cart/index.html.twig [hunk 2]
```diff
@@ -0,0 +1,38 @@
+{% extends 'layout.html.twig' %}
+
+{% from "_macros/productCard.html.twig" import productCard %}
+
+{% block content %}
+	<main class="cart container">
+		<h1 class="text-center">Mon Panier</h1>
+		<div class="productsCart">
```
Comment (Benja-san) [inline src/View/Cart/index.html.twig]: shuld be an ul tag since it will be a product list ?

File: src/View/Cart/index.html.twig [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+			{% if not (products is empty) %}
+
+				{% for product in products %}
+					<div class="products-cart">
```
Comment (Benja-san) [inline src/View/Cart/index.html.twig]: li instead of div ?

**Review by Benja-san — APPROVED** [review]
Change div tags inside Cart/index.html into ul / li and you can merge

**Review by n3en — APPROVED** [review]

**Pull request merged**